tests/features/owner/owner.moderation.feature

```diff
@@ -0,0 +1,136 @@
+    Then I should see the heading "Edit Owner EU healthy movement"
+    And the "State" select available options should be "Validated, Deletion request"
+    And the option "Validated" should be selected
+    And I should not see the link "Delete"
+    When I fill in "Name" with "EU healthy group"
+    And I press "Save"
+    Then I should see the heading "EU healthy group"
+
+    # Request an update as moderator: the chosen owner type is wrong.
+    When I am logged in as a moderator
+    And I go to the homepage of the "EU healthy group" owner
+    And I click "Edit" in the "Entity actions" region
+    Then I should see the heading "Edit Owner EU healthy group"
+    And the "State" select available options should be "Validated, In assessment, Deletion request"
+    And the option "Validated" should be selected
+    # Deletion should not be possible as the owner entity is referenced by the collection.
+    And I should not see the link "Delete"
+    # Change the state and save.
+    When I select "In assessment" from "State"
+    And I press "Save"
+    Then I should see the heading "EU healthy group"
+
+    # Another authenticated user should not be allowed to edit the owner entity.
+    When I am logged in as an "authenticated user"
+    And I go to the homepage of the "EU healthy group" owner
+    Then I should not see the link "Edit" in the "Entity actions" region
+
+    # The original owner creator is allowed to update the entity with the
+    # requested changes.
+    When I am logged in as "Raeburn Hibbert"
+    And I go to the homepage of the "EU healthy group" owner
+    And I click "Edit" in the "Entity actions" region
+    Then I should see the heading "Edit Owner EU healthy group"
+    And the "State" select available options should be "In assessment"
+    And I should not see the link "Delete"
+    # Do the changes.
+    When I check the box "Non-Governmental Organisation"
+    And I uncheck the box "Academia/Scientific organisation"
+    And I press "Save"
+    Then I should see the heading "EU healthy group"
+
+    # The moderator approves the changes.
+    When I am logged in as a moderator
+    And I go to the homepage of the "EU healthy group" owner
+    And I click "Edit" in the "Entity actions" region
+    Then the "State" select available options should be "In assessment, Validated"
+    And the option "In assessment" should be selected
+    When I select "Validated" from "State"
+    And I press "Save"
+    Then I should see the heading "EU healthy group"
+
+    # The owner asks for deletion.
+    When I am logged in as "Raeburn Hibbert"
+    And I go to the homepage of the "EU healthy group" owner
+    And I click "Edit" in the "Entity actions" region
+    Then the "State" select available options should be "Validated, Deletion request"
+    And the option "Validated" should be selected
+    When I select "Deletion request" from "State"
+    And I press "Save"
+    Then I should see the heading "EU healthy group"
```

Comment:
At this point something is wrong. Mr. Hibbert cannot edit the owner anymore now. So if he changes his mind he cannot undo the deletion request.